app/Casts/StoreScraperStrategySetCast.php

```
@@ -0,0 +1,41 @@
+     */
+    public function get(Model $model, string $key, mixed $value, array $attributes): StoreScraperStrategySetDto
+    {
+        $decoded = is_string($value) ? json_decode($value, true) : $value;
+
+        return StoreScraperStrategySetDto::fromArray(is_array($decoded) ? $decoded : null);
+    }
+
+    /**
+     * @param  array<string, mixed>  $attributes
+     */
+    public function set(Model $model, string $key, mixed $value, array $attributes): ?string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$model'. (undefined)

(UnusedFormalParameter)